lambda/rest-endpoints/src/app.ts

```diff
@@ -142,4 +144,15 @@ router.get('/api/:region/:variant/hybrid-curation.json', (req, resp) => {
 		});
 });
 
+router.post(
+	'/api/check-template',
+	(req: Request<object, object, RecipeV3>, resp) => {
```

Comment:
- Can we have example POST request to see this `[req](req: Request<object, object, RecipeV3>)` structure please? 
- Do we need to pass whole json value for recipeV3 here?
- Can we have provision of passing URL of any recipe instead?